samples/caf_sensor_manager/README.rst

```diff
@@ -70,7 +67,11 @@ After programming the sample to your development kit, test it by performing the
 Dependencies
 ************
 
-This sample uses the following |NCS| subsystems:
+This sample uses the following |NCS| libraries:
 
 * :ref:`caf_sensor_manager`
 * :ref:`caf_sensor_data_aggregator`
+* :ref:`wave_gen`
+* :ref:`app_event_manager`
+This sample uses the following |NCS| drivers:
+* :ref:`sensor_sim`
```

Comment:
Add blank lines to avoid breaking the list:
```suggestion
* :ref:`app_event_manager`

This sample uses the following |NCS| drivers:

* :ref:`sensor_sim`
```